src/index.js

```diff
@@ -110,6 +112,12 @@ export default (function create(/** @type {Options} */ defaults) {
 	// 3b smaller:
 	// redaxios.spread = (fn) => /** @type {any} */ (fn.apply.bind(fn, fn));
 
+	/** @public */
+	redaxios.interceptors = {
+		request: new Interceptor(),
+		response: new Interceptor()
+	};
```

Comment:
Using an imported class here adds a bunch of module and setup overhead and makes it so the registry of interceptors can't be a simple local variable.